src/plugins/intel_gna/tests/unit/transformations/gna_pwl.cpp

```diff
@@ -9,19 +9,34 @@
 #include <transformations/init_node_info.hpp>
 #include <type_traits>
 
+#include "common/numerical_utils.hpp"
 #include "common_test_utils/data_utils.hpp"
 #include "common_test_utils/ngraph_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "openvino/opsets/opset11.hpp"
+#include "openvino/runtime/intel_gna/properties.hpp"
 #include "transformations/pwl_approximation.hpp"
 
+using namespace ov::intel_gna;
 using namespace ov::intel_gna::common;
 
+constexpr double kAccuracyMaxErrorPrecentage = 0.09;
+constexpr double kPerformanceMaxErrorPrecentage = 1.0;
+
+double get_max_error_percentage(const PWLApproximationMode& mode) {
+    if (PWLApproximationMode::ACCURACY == mode) {
+        return kAccuracyMaxErrorPrecentage;
+    }
+    return kPerformanceMaxErrorPrecentage;
+}
+
 namespace {
 
 template <typename T>
 struct Function {};
 
 template <>
-struct Function<ngraph::opset9::Sigmoid> {
+struct Function<ov::opset11::Sigmoid> {
```

Comment:
It would be better to use "using namespace ov::opset11;" and then Function<Sigmoid>. It would be easier in the future to change the opset to the newest for the whole file.